### DISCUSSION TITLE
Possible to use more updated browserslist with postcss-preset-env?

### DISCUSSION THREAD

**Discussion by brycewray** [csstools/postcss-plugins#1299] [Q&A] [answered]
The most recent version of postcss-preset-env doesn’t have the most up-to-date version of browserslist, so is there a way to make postcss-preset-env detect a separate browserslist installation and rely on its data, instead of the browserslist dependency within postcss-preset-env?

**Comment by Antonio-Laguna** [comment, marked as answer]
Hey @brycewray thats a great question!

Because of how npm works and how we’ve specified the dependencies, npm will actually install the latest version in range. This means that any version after ours that isn’t breaking is a valid version and will just work/be installed in a normal installation

**Reply by brycewray** [reply]
@Antonio-Laguna Thanks! Very helpful info. Tried in vain to find it through searching — well, everywhere — and had no luck.
